### PR TITLE
Add get local layer name helper

### PR DESCRIPTION
**Pull request by alastairdglennie**

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp [hunk 1]
```diff
@@ -266,6 +266,15 @@ UAbstractLBStrategy* ULayeredLBStrategy::GetLBStrategyForVisualRendering() const
 	return LayerNameToLBStrategy[SpatialConstants::DefaultLayer];
 }
 
+const FName& ULayeredLBStrategy::GetLocalLayerName() const
+{
+	checkf(IsReady(), TEXT("Tried to get worker layer name but it was assigned."));
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp:271]:
```suggestion
	checkf(IsReady(), TEXT("Tried to get worker layer name but it was not assigned."));
```

File: SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp [hunk 2]
```diff
@@ -266,6 +266,15 @@ UAbstractLBStrategy* ULayeredLBStrategy::GetLBStrategyForVisualRendering() const
...
+
+	const FName* LocalLayerName = VirtualWorkerIdToLayerName.Find(LocalVirtualWorkerId);
+	checkf(LocalLayerName != nullptr, TEXT("Load balancing strategy didn't contain mapping between virtual worker ID to layer name."), LocalVirtualWorkerId);
+	return *LocalLayerName;
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp:275]: Should we perhaps enforce returning `NAME_None` instead of nullptr?

**Comment by m-samiec** [comment]
Close this out @alastairdglennie ?

**Comment by alastairdglennie** [comment]
> Close this out @alastairdglennie ?

I'll close you out, sunshine

**Pull request merged**